modules/nf-core/samtools/cramsize/main.nf

```diff
@@ -0,0 +1,35 @@
+process SAMTOOLS_CRAMSIZE {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::samtools=1.17"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/samtools:1.17--h00cdaf9_0':
+        'biocontainers/samtools:1.17--h00cdaf9_0' }"
+
+    input:
+    tuple val(meta), path(cram)
+
+    output:
+    tuple val(meta), path("*.size"), emit: size
+    path "versions.yml"            , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+
```

Comment:
```suggestion
    def prefix = task.ext.prefix ?: "$meta.id"
```
You should use prefix for your output files. This way the module is set up the most flexible

Reply:
Thanks a lot for your comments and suggestions @nvnieuwk!!!